tests/integration/test_tag.sh

```diff
@@ -48,6 +48,7 @@ NAME="TEST3"
 ${ATOMIC} images tag atomic-test-1:latest at1:latest
 ${DOCKER} inspect at1:latest 1>/dev/null 
 
-# Tag an ostree image
-${ATOMIC} pull --storage ostree docker:atomic-test-1:latest
-${ATOMIC} images tag --storage ostree atomic-test-1:latest at1:latest
+# Check that the tagged image is being displayed
+${ATOMIC} images list | grep "at1"
+${ATOMIC} images list --json | grep "at1"
+${ATOMIC} images list -f repo=at1 | grep "at1"
```

Comment:
While we are at it, should we have a test_images_list.sh integration test that exercises and checks for some things?

and then perhaps a unittest test that could check things like headers output, no images in backends, different backends, filtering ... this could be done with mock.